public/app/plugins/panel/canvas/module.tsx

```diff
@@ -59,7 +59,15 @@ export const plugin = new PanelPlugin<Options>(CanvasPanel)
         },
       },
       [FieldConfigProperty.Links]: {
+        settings: {
+          showOneClick: false,
+        },
+      },
+      [FieldConfigProperty.Actions]: {
         hideFromDefaults: true,
```

Comment:
not needed any more, but we'll need to flip this to false in next PR that adds support